README.md

```diff
@@ -1,36 +1,50 @@
 [OpenNMS][]
 ===========
 
-[OpenNMS][] is the world's first enterprise grade network management application platform developed under the open source model.
+[OpenNMS][] is an open-source network monitoring platform that helps you visualize and monitor everything on your local and distributed networks. OpenNMS offers comprehensive fault, performance, and traffic monitoring, and alarm generation in one place. Highly customizable and scalable, OpenNMS integrates with your core business applications and workflows.
 
-Well, what does that mean?
 
-*	World's First
+Features
+---------
 
-	The OpenNMS Project was started in July of 1999 and registered on SourceForge in March of 2000. It has years of experience on the alternatives.
+* **Full inventory management**
 
-*	Enterprise Grade
+	Flexible provisioning system provides many ways to interoperate with configuration management systems.
 
-	It was designed from "day one" to monitor tens of thousands to ultimately unlimited devices with a single instance. It brings the power, scalability and flexibility that enterprises and carriers demand.
+* **Extensive data collection**
 
-*	Application Platform
+	Works with many industry-standard data collection protocols (14) with no need to write or maintain third-party plugins: SNMP, JSON, WinRM, XML, SQL, JMX, SFTP, FTP, JDBC, HTTP, HTTPS, VMware, WS-Management, Prometheus.
 
-	While OpenNMS is useful "out of the box," it is designed to be highly customizable to create an unique and integrated management solution.
+* **Robust traffic management**
 
-* Open Source
+	Supports four flow protocols (NetFlow v.5/9, IPFIX, sFlow). 300,000+ flows/sec. Deep-dive analysis, enterprise reporting.
 
-	OpenNMS is 100% Free and Open Source software, with no license fees, software subscriptions or special "enterprise" versions.
+* **Digital experience monitoring**
 
-Installing OpenNMS
+	 Use the OpenNMS Minion to monitor a service’s latency and availability from different perspectives.
+
+* **Robust configuration**
+
+	Configure most features through the web UI or XML scripting, including thresholding, provisioning, event and flow management, service monitoring, and performance measurement.
+
+* **Scalability**
+
+	Monitor hundreds of thousands of nodes using a distributed and tiered system.
```

Comment:
IMHO, this here is overpromising. Instead of throwing out a number, I would highlight the scaling points. Sentinels on the flow persisting side, message broke from embedded ActiveMQ to Kafka, and Flow, BMP, SNMP trap, Syslog ingest with Minions.

Reply:
Not sure I understand the context for the message brokers, so changes might not work.